functions/Get-DbaAgentAlert.ps1

```diff
@@ -49,6 +57,7 @@ function Get-DbaAgentAlert {
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]
         $SqlCredential,
+        [string[]]$Name,
```

Comment:
This should be called `Alert` as it is the name of the SMO object we are returning. Just like with `Get-DbaAgentSchedule` and `Get-DbaAgentOperator` they are named `Schedule` and `Operator` respectfully.

As well, an additional parameter for `ExcludeAlert` should be added to return all but the provided list as you have done in other command(s).

Reply:
Sounds good to me. I've done all of that and while I was at it also added support for regexes on both include and exclude.

Reply:
The pattern support can be included within each original Alert and ExcludeAlert. We do not need additional parameters for that, causes more confusion on user side. 

I would just update the help to state that each parameter also accepts wildcards or regex patterns.